### DISCUSSION TITLE
Copilot only returns this Error now Request Error: Stream closed with error code NGHTTP2_INTERNAL_ERROR

### DISCUSSION THREAD

**Discussion by sergiyvelichkevych** [community/community#56248] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot only returns this Error now Request Error: Stream closed with error code NGHTTP2_INTERNAL_ERROR

Please see details in
https://github.com/orgs/community/discussions/55575

Same issue with VCS remote-SSH. 

[INFO] [default] [2023-05-17T17:27:07.921Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[ERROR] [default] [2023-05-17T17:27:17.981Z] Request Error: Stream closed with error code NGHTTP2_INTERNAL_ERROR
[ERROR] [ghostText] [2023-05-17T17:27:17.983Z] Error on ghost text request FetchError: Stream closed with error code NGHTTP2_INTERNAL_ERROR

Still doesn't work.  Please investigate and fix.

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/55575

**Comment by davecheney** [comment]
@sergiyvelichkevych thank you for raising this issue. The development team are aware of the issue and are investigating. I suggest you contact support, https://support.github.com/contact who can escalate your request.
